CHANGELOG.md

```diff
@@ -1,8 +1,83 @@
+# v0.39.0 Release - 09/26/2019
+
+*Note*: This release comes with a new config version `v1beta15`.
+        To upgrade your `skaffold.yaml`, use `skaffold fix`. If you don't upgrade, skaffold will auto-upgrade in memory as best it can, and print a warning message.
+        See [deprecation-policy.md](/deprecation-policy.md) for details on what beta means.
+        The env vars `DIGEST`, `DIGEST_HEX` and `DIGEST_ALGO` are now fail in `envTemplate` fields. 
+
+New Features:
+
+* Optional pull secret for Kaniko [#2910](https://github.com/GoogleContainerTools/skaffold/pull/2910)
+* Add Jib-Gradle support for Kotlin buildscripts [#2914](https://github.com/GoogleContainerTools/skaffold/pull/2914)
+* Add graceful termination for custom builders [#2886](https://github.com/GoogleContainerTools/skaffold/pull/2886)
+* Add docs and tutorial for buildpacks [#2879](https://github.com/GoogleContainerTools/skaffold/pull/2879)
+* kustomize build args [#2871](https://github.com/GoogleContainerTools/skaffold/pull/2871)
```

Comment:
Can you grab the release note from this PR? I think i filled in the release note section here which is more in details

Reply:
I don't see the release notes on this one

Reply:
hmm looks like i put release notes on another PR #2895 
This is also a config change can you change this to 
```
You can now pass `buildArgs` to kustomize deployer in `KustomizeDeploy` config.
```